mix.exs

```diff
@@ -30,7 +30,8 @@ defmodule Scholar.MixProject do
   defp deps do
     [
       {:ex_doc, "~> 0.30", only: :docs},
-      {:nx, "~> 0.6", override: true},
+      # {:nx, "~> 0.6", override: true},
+      {:nx, github: "elixir-nx/nx", sparse: "nx", override: true},
```

Comment:
We can continue using 0.6, no?

Reply:
Not really, since in the main there is a fix for `Nx.clip` bug with infinities. In Isotonic regression, this is a default case, unfortunately.

Reply:
I see!